### ISSUE TITLE
Doesn't work with testing-library-selector 

### ISSUE THREAD

**Issue by AlexKrupko** [testing-library/eslint-plugin-jest-dom#371]
The following code doesn't warn any errors: 
```tsx
import {byRole} from 'testing-library-selector';

const ui = {
    input: byRole('textbox');
};

test('test', () => {
    render(<input />);
    expect(ui.input.get()).toHaveAttribute('disabled'); // <--- no errors here 
});
```

Is it possible to use these rules when `screen.getBy...` methods are not used directly as an element?

**Comment by G-Rath** [comment]
No, I don't see a good way we could support this given you're importing from `testing-library-selector`, the import is `ByRole` (rather than `getByRole`), and the argument to `expect` is `ui.input.get()` which is very generic - it might be more possible if we TypeScript based, but then it would require everyone to use TypeScript.

**Comment by AlexKrupko** [comment]
Yes, I thought this is impossible. So, we'll have to exclude this plugin from the project then. 
Also, I don't think that Typescript can help somehow here... 
Anyway, thank you for the response @G-Rath.
